TransformationSystem/Client/TaskManager.py

```diff
@@ -566,28 +569,37 @@ def __prepareTransformationTasks( self, transBody, taskDict, owner, ownerGroup,
     """ Prepare transformation tasks with a job object per task
     """
     method = '__prepareTransformationTasks'
+    startTime = time.time()
+    oJobTemplate = self.jobClass( transBody )
+    oJobTemplate.setOwner( owner )
+    oJobTemplate.setOwnerGroup( ownerGroup )
+    oJobTemplate.setOwnerDN( ownerDN )
+    site = oJobTemplate.workflow.findParameter( 'Site' ).getValue()
+    jobType = oJobTemplate.workflow.findParameter( 'JobType' ).getValue()
+    templateOK = False
+    outputTime = 0.
     for taskID, paramsDict in taskDict.iteritems():
       # Create a job for each task and add it to the taskDict
-      oJob = self.jobClass( transBody )
-      site = oJob.workflow.findParameter( 'Site' ).getValue()
+      if not templateOK:
+        templateOK = True
```

Comment:
in order to add missing information in the "template" we need to get the first job's information